ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/143__dsc_negative_dependant_operators_not_installed.robot

```diff
@@ -124,6 +148,17 @@ Reinstall Service Mesh Operator And Recreate DSC And DSCI
     Set Service Mesh State To Managed And Wait For CR Ready
     ...           ${SERVICE_MESH_CR_NAME}    ${SERVICE_MESH_CR_NS}    ${SERVICE_MESH_OPERATOR_NS}
 
+Reinstall Serverless Operator And Recreate DSC And DSCI
+    [Documentation]    Reinstalls Service Mesh operator and waits for the Service Mesh Control plane to be created
+    Remove DSC And DSCI Resources
+    Install Serverless Operator Via Cli
+    Apply DSCInitialization CustomResource    dsci_name=${DSCI_NAME}
+    Wait For DSCInitialization CustomResource To Be Ready    timeout=180
+    Apply DataScienceCluster CustomResource    dsc_name=${DSC_NAME}
+    Wait For DataScienceCluster CustomResource To Be Ready   timeout=600
+    Set Service Mesh State To Managed And Wait For CR Ready
+    ...           ${SERVICE_MESH_CR_NAME}    ${SERVICE_MESH_CR_NS}    ${SERVICE_MESH_OPERATOR_NS}
+
```

Comment:
Service Mesh is Managed when creating DSCI with default values. May not need to attempt to do so again.